oauth2client/service_account.py

```diff
@@ -461,3 +465,100 @@ def create_delegated(self, sub):
         result._private_key_pkcs12 = self._private_key_pkcs12
         result._private_key_password = self._private_key_password
         return result
+
+
+class JWTAccessCredentials(GoogleCredentials):
+    """Self signed JWT credentials.
+
+    Makes an assertion to server using a self signed JWT from service account
+    credentials.  These credentials do NOT use OAuth 2.0 and instead authenticate
+    directly.
+    """
+    MAX_TOKEN_LIFETIME_SECS = 3600
+    """Max lifetime of the token (one hour, in seconds)."""
+
+    def __init__(self, service_account_creds, additional_claims=None, access_token=None,
+                 token_expiry=None):
+        super(JWTAccessCredentials, self).__init__(access_token, None, None, None, token_expiry,
```

Comment:
Why `GoogleCredentials` as the superclass? Why not just `oauth2client.client.Credentials`?